config/prod/tom-ncats.yaml

```diff
@@ -1,6 +1,13 @@
 # Provision an EC2 instance
-template_path: "remote/managed-ec2-linux-v1.j2"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/managed-ec2-linux-v1.j2"
 stack_name: "tom-ncats"
+stack_tags:
+  Department: "CompOnc"
+  Project: "dream"
+  OwnerEmail: "thomas.yu@sagebase.org"
+  CostCenter: "Genie-AACR / 312000"
```

Comment:
How did you determine this CostCenter?  I can't infer any particular CostCenter from "dream".

Reply:
just a guess.  @thomasyu888  can you please verify the most appropriate value for this?

Reply:
ping @thomasyu888

Reply:
@zaro0508 I think this is fine.